The last pull request (#34) did not activate the *Publish to PyPi* workflow

LuisOlivaresJ opened the pull request:
Hi,

I am suspecting that the Action did not work because the **main** branch name. Additionally, we need to change the version on every pull request to OMG_master, otherwise packaging to PyPi will show an error because the version number already exist.

Cheers,
Luis

LuisOlivaresJ committed:
The last pull request (#34) did not activate the *Publish to PyPi* workflow. I am suspecting that it is because the main branch name. Additionally, we need to change the version on every pull request to OMG_main branch, otherwise packaging to PyPi will show an error because the version number already exist.

petruong commented on pyproject.toml:
```diff
@@ -5,7 +5,7 @@ build-backend = "hatchling.build"
 name = "omg_dosimetry"
 
-version = "1.5.5"
+version = "1.6.0"
```
So, from what I understand: is this the version line that has to be updated each pull request?

jfcabana replied:
Correct. Pull requests to Master should have the version number incremented. We're using the recommended "semantic versioning" (https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#semantic-versioning-preferred).

We implemented automatic actions that are triggered with every pull request to master. Documentation on ReadTheDocs is updated automatically, and the new version is built and published on pypi.

petruong commented:
Would it be reasonable to include a few spelling error corrections in the README.rst for this pull request?
If so, I can add the changes in a commit prior to the merging of this pull request.

jfcabana commented:
> Would it be reasonable to include a few spelling error corrections in the README.rst for this pull request? If so, I can add the changes in a commit prior to the merging of this pull request.

Why not!

petruong committed:
Update README.rst

- Corrected a few spelling/grammatical errors. 
- Standardized/capitalized sub-titles

petruong commented:
> > Would it be reasonable to include a few spelling error corrections in the README.rst for this pull request? If so, I can add the changes in a commit prior to the merging of this pull request.
> 
> Why not!

Updated on my end.

LuisOlivaresJ commented:
All right to me.
